Plan/src/main/java/com/djrapitops/plan/system/database/databases/sql/patches/Patch.java

```diff
@@ -67,32 +74,45 @@ public Boolean processResults(ResultSet set) throws SQLException {
     }
 
     protected boolean hasColumn(String tableName, String columnName) {
-        return usingMySQL ?
-                query(new QueryStatement<Boolean>("SELECT * FROM information_schema.COLUMNS" +
-                        " WHERE TABLE_NAME=? AND COLUMN_NAME=? AND TABLE_SCHEMA=?") {
-                    @Override
-                    public void prepare(PreparedStatement statement) throws SQLException {
-                        statement.setString(1, tableName);
-                        statement.setString(2, columnName);
+        if (usingMySQL) {
```

Comment:
Cognitive complexity of this method was already high, but now it is reaching the limits where it will be a problem.

See if this can be refactored into simpler form after `DBType` enum has been introduced.

Reply:
Unsure how this can be simplified, even after the introduction of `DBType`.
Only thing I can think of is splitting the method into two, one for MySQL compatible ones and the other for not.